#103: Added option to toggle TLS cert validation (#123)
exasol/transformers-extension: doc/user_guide/user_guide.md

      --bucketfs-port <BUCKETFS_PORT> \
      --bucketfs-user <BUCKETFS_USER> \
      --bucketfs-password <BUCKETFS_PASSWORD> \
      --bucketfs-use-https <USE_HTTPS_BOOL> \
      --bucket <BUCKETFS_NAME> \
      --path-in-bucket <PATH_IN_BUCKET> \
      --language-alias <LANGUAGE_ALIAS> \ 
      --version <RELEASE_VERSION> \
      --ssl-cert-path <ssl-cert-path> \
      --use-ssl-cert-validation \
      --no-use-ssl-cert-valiation
  ```

> The `--ssl-cert-path` is optional if your cert is in an atypical path. The option `--use-ssl-cert-validation` 

Reviewer: ```suggestion
The `--ssl-cert-path` is optional if your certificate is not in the OS truststore. The option `--use-ssl-cert-validation` 
```

      --bucketfs-port <BUCKETFS_PORT> \
      --bucketfs-user <BUCKETFS_USER> \
      --bucketfs-password <BUCKETFS_PASSWORD> \

>       --bucketfs_use-https <USE_HTTPS_BOOL> \

Reviewer: use - instead of underscore

      --bucket <BUCKETFS_NAME> \
      --path-in-bucket <PATH_IN_BUCKET> \
      --language-alias <LANGUAGE_ALIAS> \ 
      --version <RELEASE_VERSION> \
      --ssl_cert_path <SSL_CERT_PATH> \

>       --use_ssl_cert_validation <USE_CERT_VALIDATION_BOOL>

Reviewer: use - instead of _

>       --use_ssl_cert_validation <USE_CERT_VALIDATION_BOOL>

Reviewer: A common pattern for bool CLI is `--ssl-cert-validation` and `--no-ssl-cert-valiation` without a value. Click can do this.

  ```

> The ssl_cert_path is optional if your cert is in an atypical path. The use_ssl_cert_validation is True by default. 

Reviewer: ```suggestion
The `--ssl-cert-path` is optional if your cert is in an atypical path. The option `--ssl-cert-validation` is the default, you can disable it with `--no-ssl-cert-validation`.
```

> Use caution if you want to turn this of as it potentially lowers security of you Exasol Database.

Reviewer: ```suggestion
Use caution if you want to turn certificate validation off as it potentially lowers the security of your Database connection.
```

    --db-user <DB_USER> \
    --db-pass <DB_PASSWORD> \
    --schema <SCHEMA> \

>     --language-alias <LANGUAGE_ALIAS> \

Reviewer: ```suggestion
    --language-alias <LANGUAGE_ALIAS>
```
